src/pydata_sphinx_theme/theme/pydata_sphinx_theme/sections/sidebar-secondary.html

```diff
@@ -1,6 +1,6 @@
-{% if theme_secondary_sidebar_items -%}
+{% if secondary_sidebar_items -%}
 <div class="sidebar-secondary-items sidebar-secondary__inner">
-{% for toc_item in theme_secondary_sidebar_items %}
+{% for toc_item in secondary_sidebar_items %}
```

Comment:
I often forget when we need to use the `theme_` prefix and when we don't. I see it's no longer needed here because in the new `set_secondary_sidebar_items` function you're creating a new entry in `context` without the `theme_` prefix. Can you add a comment here with a cross-ref to `utils.py` so the next dev knows where to look if they need to modify something?

Reply:
You're absolutely right. Maybe there's a better choice of variable name here? Anyway, I added a note.